resources/js/Pages/Vault/Settings/Partials/TabVisibility.vue

```diff
@@ -0,0 +1,132 @@
+<script setup>
+import { useForm } from '@inertiajs/inertia-vue3';
+import { onMounted } from 'vue';
+
+const props = defineProps({
+  data: Object,
+});
+
+const form = useForm({
+  show_group_tab: false,
+  show_tasks_tab: false,
+  show_files_tab: false,
+  show_journal_tab: false,
+});
+
+onMounted(() => {
+  form.show_group_tab = props.data.visibility.show_group_tab;
+  form.show_tasks_tab = props.data.visibility.show_tasks_tab;
+  form.show_files_tab = props.data.visibility.show_files_tab;
+  form.show_journal_tab = props.data.visibility.show_journal_tab;
+});
+
+const update = () => {
+  axios.put(props.data.url.update_tab_visibility, form);
```

Comment:
when using a `useForm`, it makes more sense to use it to send the request, like
```suggestion
  form.put(props.data.url.update_tab_visibility);
```
(but the response must be an Inertia Request ...)

Reply:
fixed